lib/rubygems/specification.rb

```diff
@@ -2097,6 +2097,9 @@ def initialize_copy(other_spec)
         raise e
       end
     end
+
+    @required_ruby_version = other_spec.required_ruby_version.dup
+    @required_rubygems_version = other_spec.required_rubygems_version.dup
```

Comment:
We should probably tweak the documentation of this method now to match the new implementation (it's not only "array attributes" now).

Reply:
Done!